app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ActiveRecord::Base
   has_secure_password
-  validates :email, presence: true, uniqueness: true
+  validates :email, uniqueness: true
+  validates_presence_of :first_name,
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.